orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/AmazonServerGroupCreator.groovy

```diff
@@ -132,7 +142,7 @@ class AmazonServerGroupCreator implements ServerGroupCreator, DeploymentDetailsA
 
   def allowLaunchOperations(Map createServerGroupOp) {
     def ops = []
-    if (createServerGroupOp.availabilityZones && createServerGroupOp.credentials != defaultBakeAccount) {
+    if (cloudProvider == 'aws' && createServerGroupOp.availabilityZones && createServerGroupOp.credentials != defaultBakeAccount) {
```

Comment:
This class is extended by the `TitusServerGroupCreator`, which sets `cloudProvider` to `titus`